### PR TITLE
search by keyword api and swagger update

### PR DESCRIPTION
**Pull request by SwordFaith**
Test case:
http://localhost:3000/items?keyword=DDP
http://localhost:3000/items?keyword=ddp
http://localhost:3000/items?keyword=Pytorch
http://localhost:3000/items?keyword=pytorch
http://localhost:3000/items?keyword=PyTorch
http://localhost:3000/items?keyword=Keras
http://localhost:3000/items?keyword=keras
http://localhost:3000/items?keyword=OpenPAI
http://localhost:3000/items?
http://localhost:3000/items?keyword=null

### CODE REVIEW

File: rest_server/marketplace_api.yaml [hunk 1]
```diff
@@ -450,29 +454,26 @@ components:
...
           type: string
         description:
           type: string
-        content:
-          oneOf:
-            - $ref: "#/components/schemas/Data"
-            - $ref: "#/components/schemas/Docker"
-            - $ref: "#/components/schemas/Model"
-            - $ref: "#/components/schemas/Job"
+        status:
```
Comment (debuggy) [inline rest_server/marketplace_api.yaml:461]: Do we have status property now?

File: rest_server/src/models/marketplace_item.js [hunk 2]
```diff
@@ -62,11 +63,18 @@ class MarketplaceItem {
       if (type && type !== 'all') {
         filterStatement.type = type;
       }
+      if (keyword) {
+        filterStatement[Op.or] = [
+          { name: { [Op.substring]: keyword } },
```
Comment (debuggy) [inline rest_server/src/models/marketplace_item.js:68]: what if key word is null

File: rest_server/marketplace_api.yaml (hunk 1 above)
Comment (SwordFaith) [reply rest_server/marketplace_api.yaml:461]: in current model, yes

File: rest_server/src/models/marketplace_item.js (hunk 2 above)
Comment (SwordFaith) [reply rest_server/src/models/marketplace_item.js:68]: keyword=null return same length as query with no "keyword", but may it better to filter "null“ out ?

Comment (debuggy) [reply rest_server/src/models/marketplace_item.js:68]: It is ok if there is not error throwed

**Pull request closed without merging**